src/binary-reader-nop.h

```diff
@@ -252,9 +252,13 @@ class BinaryReaderNop : public BinaryReaderDelegate {
   Result OnMemoryGrowExpr() override { return Result::Ok; }
   Result OnMemoryInitExpr(Index segment_index) override { return Result::Ok; }
   Result OnMemorySizeExpr() override { return Result::Ok; }
-  Result OnTableCopyExpr() override { return Result::Ok; }
+  Result OnTableCopyExpr(Index dst_index, Index src_index) override {
```

Comment:
This is formatted differently from the surrounding code?

Reply:
I think its just because it overflowed 80 chars.  I ran clang format and this stayed the same (but other things I missed, so thanks!)